packages/fds-components/src/style/buttons/buttonGroup.css

```diff
@@ -0,0 +1,70 @@
+:root {
+  --button-group-timeout: 250ms;
+}
+
+.btn-group {
+  display: inline-flex;
+}
+
+.btn-group > .group-btn {
+  margin-left: -1px;
+  position: relative;
+  border-radius: 0;
+}
+
+.btn-group > .group-btn:focus {
+  z-index: 2;
+}
+
+.btn-group > .group-btn:first-child {
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+  margin-left: 0;
+}
+
+.btn-group > .group-btn:last-child {
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+
+.group-btn {
+  border: 1px solid #B3B5B6;
+  border-radius: 5px;
+  color: var(--color-slate);
+  padding: 12px;
+  font-size: 12px;
+  text-transform: none;
+  line-height: normal;
+  display: inline-flex;
+  align-items: center;
+  user-select: none;
+  transition: background-color var(--button-group-timeout) var(--motion-easing-default),
+    box-shadow var(--button-group-timeout) var(--motion-easing-default),
+    border var(--button-group-timeout) var(--motion-easing-default);
+}
+
+.group-btn .fds-icon {
+  fill: currentColor;
+}
+
+
+.group-btn:focus {
+  background-color: #D3DDE4;
+}
+
+.group-btn:hover {
+  text-decoration: none;
+  background-color: #DEE5EB;
+}
+
+.group-btn.active {
```

Comment:
Should active buttons have bold text?

Reply:
According to Sketch they should. Good catch.

Though one thing I worry about here: if we bold the text, the buttons shift ever so slightly, because bold makes the font width increase. Are we ok with that?